addpeer.go

```diff
@@ -5,8 +5,8 @@ import (
 	"fmt"
 	"log"
 
-	inet "github.com/libp2p/go-libp2p-net"
-	pstore "github.com/libp2p/go-libp2p-peerstore"
+	inet "gx/ipfs/QmPjvxTpVH8qJyQDnxnsxF9kv9jezKD1kozz1hs3fCGsNh/go-libp2p-net"
```

Comment:
We might need to [`gx-uw`](https://github.com/ethresearch/sharding-p2p-poc/blob/master/Makefile#L42) to recover the original package source as the [`go get`](https://github.com/ethresearch/sharding-p2p-poc/blob/master/.travis.yml#L22) command could not recognize the gx source.

Reply:
Ok, thanks for the tip